src/Microbuild.Settings.targets

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It is our own helper tasks that do some processing before Microbuild targets, like zipping/unzipping VS templates for localization, which msbuild doesn't do by default.